app/main/db/models.py

```diff
@@ -0,0 +1,72 @@
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import Boolean, DateTime, ForeignKey, String, Text
+from sqlalchemy.dialects.postgresql import UUID
+
+db = SQLAlchemy()
+
+
```

Comment:
I've used flask migrate before which yes does use Alembic under the hood but abstracts away some of the lower level stuff and should be easier to work with. 

Ideally there should be one source of truth that is then replicated elsewhere which sounds like we definitely do need an API to handle this.

Reply:
yeah, if we decide the db is defined outside the webapp then I think whatever that is it needs an api around it for us to interact with.

Down the line, we will be making calls to external apis anyway that once we are interacting with axiel collections etc. rather than reading from a db in the webapp itself but since we already have a flask app we get an api for free rather than needing to create a new microservice essentially which might not be needed down the road?

Reply:
I will review it , based on the discussion the converstation has been marked as resolved